tests/PSRule.Rules.Azure.Tests/Resources.VirtualNetwork.json

```
@@ -1731,10 +1736,7 @@
               }
             ],
             "privateIPAddressVersion": "IPv4"
-          },
-          "zones": [
-            "3"
-          ]
+          }
```

I think we should still check and fail for the case where a zonal LB is configured.


```suggestion
          },
          "zones": [
            "3"
          ]
```